Wikipedia/Code/WMFSearchLanguagesBarViewController.swift

```diff
@@ -71,7 +76,7 @@ class WMFSearchLanguagesBarViewController: UIViewController, WMFPreferredLanguag
 
     private func selectFirstLanguageIfNoneSelectedOrIfFirstLanguageHasChanged(){
```

Comment:
You shouldn't need this method anymore… just call setCurrentlySelectedLanguageToButtonLanguage() directly from viewDidAppear